### DISCUSSION TITLE
Github Campus Expert Results?

### DISCUSSION THREAD

**Discussion by fykaa** [community/community#36473] [GitHub Education] [answered]
It's been around 3 weeks since we submitted our github campus expert video resume, but have no updates on the same. Does anyone here applied to Aug 2022 GCE program and got a mail yet?

**Comment by melvincwng** [comment]
@fykaa Hi Fyka, understand you posted this 7 days ago, hence would like to check again if you have received any update from the GitHub Education Team?

Would @samus-aran or @juanpflores be able to assist/check on this? If unable to, pls kindly disregard this ping/comment.

Appreciate any response on this, thanks 👍

**Comment by fykaa** [comment, marked as answer]
Hello, @melvincwng .
I have yet to receive an update.
I queried many individuals on the GCE team, and they claimed it may arrive this week.

Hope this answers your question.

**Reply by melvincwng** [reply]
@fykaa Hey Fyka, thanks for your response!

All the best in the video applications for GCE Aug 2022 :)

**Reply by melvincwng** [reply]
Hi @fykaa, have you currently received any updates on your end?

**Reply by fykaa** [reply]
Hey @melvincwng ! Nope :(

**Reply by melvincwng** [reply]
Hi @elisemoe, would you be able to kindly assist or check the application status for the GCE Aug program for both me and @fykaa?

Have already submitted a support ticket 7-8 days ago, but did not receive any reply on this as well.

The application portal does not have any updates as well.

Appreciate any response on this (if possible), thanks 👍

**Reply by lnxpy** [reply]
Same thing goes for me. Haven't received anything about the results.

**Comment by melvincwng** [comment]
Hi @fykaa & @lnxpy, I have just received an update from the GitHub Education Team via email.

Do kindly lookout for any emails in your inbox today.

All the best in your applications 🎉

Take care, and have a good day ahead 😊

**Reply by fykaa** [reply]
Heyy @melvincwng !
Happy to share that I've been selected too😊
Let's meet at the GitHub Campus Experts Onboarding Call !!!

**Reply by samus-aran** [reply]
Congratulations @fykaa & @melvincwng 👏 💃

**Reply by fykaa** [reply]
Thank you @samus-aran !

**Reply by melvincwng** [reply]
Thanks @samus-aran 🎉
